portfolio/src/main/webapp/script.js

```diff
@@ -40,3 +40,7 @@ function navbar_set_active_item(){
   else if (current_webpage[0] == 't')
     document.getElementById("series").classList.add("active");
 }
+
+function redirectFunction(){
+  window.location.pathname='/resources/resume.pdf';
+}
```

Comment:
Always add new line at the end of a file. This makes it consistent that each line in a file has always a newline char at the end.